packages/expo-dev-menu-interface/ios/DevMenuManagerProtocol.swift

```diff
@@ -10,9 +10,6 @@ public protocol DevMenuManagerProtocol {
   @objc
   var isVisible: Bool { get }
 
-  @objc
-  var delegate: DevMenuDelegateProtocol? { get set }
```

Comment:
Hmm, I'm not a big fan of this change. On one hand, it makes our codebase cleaner, but also I was added to give the ability to customize dev menu behavior by removing it we also remove that customization. 

To be fair, it wasn't used in our codebase. So maybe we can reiterate that in the future. For now, let's remove it.